src/main/java/core/basesyntax/Engine.java

```diff
@@ -1,5 +1,78 @@
 package core.basesyntax;
 
-public class Engine {
+public class Engine implements Cloneable {
+    private String model;
+    private String fuel;
+    private String row;
+    private double volume;
+    private int power;
 
+    public Engine(String model, String row, double volume, int power, String fuel) {
+        this.model = model;
+        this.row = row;
+        this.volume = volume;
+        this.power = power;
+        this.fuel = fuel;
+    }
+
+    @Override
+    public Engine clone() throws CloneNotSupportedException {
```

Comment:
```suggestion
    public Engine clone() {
```
you process exception inside of the method, we don't need to add it to the signature as well